README.md

```diff
@@ -1,5 +1,5 @@
 <p align="center">
-<img src="./assets/remix-dev-tools.png" style="display: block; margin: 0 auto;" align="middle" width="240" height="240" alt="Remix Development Tools"  />
+<img src="./assets/remix-dev-tools.png" style="display: block; margin: 0 auto;" align="middle" height="240" alt="Remix Development Tools"  />
```

Comment:
Not sure if you want this to be longer. I kept with the original heigh you had but let the length auto-figure out